astropy/units/cgs.py

```diff
@@ -108,6 +108,8 @@
 
 def_unit(['G', 'Gauss', 'gauss'], 1e-4 * si.T, namespace=_ns, prefixes=True,
          doc="Gauss: CGS unit for magnetic field")
+def_unit(['Mx', 'Maxwell', 'maxwell'], 1e-8 * si.Wb, namespace=_ns, prefixes=True,
```

Comment:
Do we want to have prefixes? I.e., is it common use to have `mMx` or `kMx`?

Reply:
At the time I was like yeah but checking some papers that use them, they keep it at Mx and they do 2x10^6 Mx instead.